datahub-web-react/src/app/entity/shared/components/styled/DeprecationPill.tsx

```diff
@@ -89,12 +114,33 @@ export const DeprecationPill = ({ deprecation, preview }: Props) => {
                         {isDividerNeeded && <ThinDivider />}
                         {deprecation?.note !== '' && <DeprecatedSubTitle>{deprecation.note}</DeprecatedSubTitle>}
                         {deprecation?.decommissionTime !== null && (
-                            <Typography.Text type="secondary">
-                                <Tooltip placement="right" title={decommissionTimeGMT}>
-                                    <LastEvaluatedAtLabel>{decommissionTimeLocal}</LastEvaluatedAtLabel>
-                                </Tooltip>
-                            </Typography.Text>
+                            <>
+                                <Typography.Text type="secondary">
+                                    <Tooltip placement="right" title={decommissionTimeGMT}>
+                                        <LastEvaluatedAtLabel>{decommissionTimeLocal}</LastEvaluatedAtLabel>
+                                    </Tooltip>
+                                </Typography.Text>
+                            </>
                         )}
+                        {isDividerNeeded && <ThinDivider />}
+                        <Button
+                            type="default"
```

Comment:
Instead of using `Button` here, can we actually just make this text with font-size 12px and a hover state with `blue[5]` as the color on hover?
Here's some screenshots of what I'm thinking:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/28656603/193611424-b1f78134-4a80-4295-b9e2-8bcd37bc096d.png">

on hover:
<img width="526" alt="image" src="https://user-images.githubusercontent.com/28656603/193611477-9dac3a7d-6859-425a-aef7-beefbcb29c2c.png">

Reply:
Checked and Fixed!!